_usage/security.md

```diff
@@ -45,9 +45,7 @@ The table below shows how Babelfish behaves for each combination:
 
 ### User management and roles
 
-By default, Babelfish comes with extra users on top of what PostgreSQL
-has to offer. Here is a list of users to be found in a fresh Babelfish
-installation: 
+Babelfis creates a number of PostgreSQL roles:
```

Comment:
Typo: Babelfish

Reply:
Oops. Fixed in 66c1e14bbb6027cf6bda77acf5c6e66779dfda5d.